packages/trpc/server/routers/viewer/bookings/get.handler.ts

```diff
@@ -862,59 +836,26 @@ async function getEventTypeIdsFromEventTypeIdsFilter(prisma: PrismaClient, event
   return eventTypeIdsFromDb;
 }
 
-async function getEventTypeIdsWhereUserIsAdminOrOwner(
```

Comment:
i like this deletion 🔥 no more need for custom logic to check resource authorization